.ci/eclipse-compiler-javac.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+ECJ_VERSION="4.6.1"
```

Comment:
However, I found way to get `4.7 `version. I use direct link on one of mirrors - 
http://ftp-stud.fht-esslingen.de/pub/Mirrors/eclipse/eclipse/downloads/drops4/R-4.7-201706120950/ecj-4.7.jar

Is it ok?